lib/internal/test_runner/runner.js

```diff
@@ -109,7 +108,7 @@ function createTestFileList(patterns) {
 
 function filterExecArgv(arg, i, arr) {
   return !ArrayPrototypeIncludes(kFilterArgs, arg) &&
-  !ArrayPrototypeSome(kFilterArgValues, (p) => arg === p || (i > 0 && arr[i - 1] === p) || StringPrototypeStartsWith(arg, `${p}=`));
+    !ArrayPrototypeSome(kFilterArgValues, (p) => arg === p || (i > 0 && arr[i - 1] === p) || StringPrototypeStartsWith(arg, `${p}=`));
```

Comment:
May I ask if the style change is necessary? (e.g. does it fail the pipeline?) if not I think would be better to put them back (maybe do a separate PR to fix the style if you have strong opinion about it) as those changes tend to make noises e.g. future trace back, backporting. (This is not blocking)

Reply:
Yes, no problem at all 🚀

Reply:
done!